tests/README.rst

```diff
@@ -80,46 +37,48 @@ Logs from the run are stored in the ``test-logs/`` folder; no other files are
 modified/touched by the container (it works on an internal copy of the host's
 anaconda directory).
 
+Interactively work inside of container
+--------------------------------------
+
 For interactively working in the container you can run::
 
-    make container-shell
+    make -f Makefile.am container-shell
 
 This command will open bash inside the container for you with mounted
 current folder at the `/anaconda` path. This is a convenient way
 how to run tests but avoid constant call of autotools and build during the
 development.
 
-Note:
-
-Please update your container from time to time to have newest dependencies.
-To do that just run the build again.
+Prepare the environment and build the sources::
 
-Run tests inside Mock
----------------------
+    ./autogen.sh
+    ./configure
+    make
```

Comment:
Won't this need the dependencies too?

Reply:
Yes, it does. But the rawhide/ELN/rhel-8 container have all the dependencies (that's the point of having the container). Note that this is now in the "Interactively work inside of container" section. That may be a bit hard to see from the diff, it may be easier to look at [the rendered README](https://github.com/martinpitt/anaconda/blob/remove-mock/tests/README.rst)?

Reply:
Ah yes, you're completely right.